packages/graphiql/src/components/DocExplorer.js

```diff
@@ -54,6 +57,29 @@ export class DocExplorer extends React.Component {
     );
   }
 
+  componentDidUpdate(prevProps) {
```

Comment:
React advises against the use of this method.

> Note:
> These methods are considered legacy and you should avoid them in new code:
> UNSAFE_componentWillUpdate()
> UNSAFE_componentWillReceiveProps()

Reply:
Hmm, is that true? It looks like componentWillUpdate is deprecated, but componentDidUpdate is not.

https://reactjs.org/docs/react-component.html#componentdidmount

Reply:
Oops, I clearly looked at the wrong part of the docs somehow :flushed: